### ISSUE TITLE
A place for brainstorming and ideas

### ISSUE THREAD

**Issue by osAIran** [tcfev/task-force-nika#12]
Hi All!

I think it would be good to have a place to brainstorm, see other people ideas, present yours and improve on existing ones.

**Comment by jinaamini** [comment]
I will start with an idea:

## A social media platform like Twitter, but not controlled by Elon musk
### Reasons
* Twitter removed IR +98 number from the registration
* People are experiencing issues with Twitter in Iran, including not being able to send direct messages
* Liking doesn't capture our feelings. We do not like seeing bad news, it would be better to add more emotions such as sadness or anger.

### Challenges
* This is a massive project and needs a lot of developers
* Trust
* Needs to be implemented fast. We don't have time
* We don't have enough personnel to do it under TCF

**Comment by zanzendegi** [comment]
We are in the very beginning of the "Twitter thingy" project so I cannot give a good estimation.

- I believe in 30 days we will have a working "temporary" backend. We then will need a mobile app and it's a good idea to start it from now.
- After our temporary backend is done, we will start the backend from scratch, and I would take about 4-6 months to complete.
- The backend and the mobile app can go in parallel. They are very much independent thanks to our temporary backend.
- I believe a working mobile app would take about 4 months but polishing the hard edges might go for a year. This really depends on the team and the operational stuff.

**Comment by jinaamini** [comment]
Is there a way to fund this project? There must be people that want to help WLF. With funding we can hire devs and maybe bootstrap in 1-2 months.

**Comment by zanzendegi** [comment]
We have GH Donations button for this repo: https://github.com/tcfev/forDem

**Comment by jinaamini** [comment]
Then we need a PE campaign to raise fund

**Comment by IranNewsWatch** [comment]
This is an important idea with two reservations:
- Cost of creating and maintaining: If it grows as much as we hope, it will be costly to maintain. The cost is not just service (coding) is also money for massive cloud-based processing
- Urgency and long-term planning: If it is not supposed to be as big as Twitter (either wanted or not to be so), why should it be a "twitter thingy"? I think it should be just a part of a central open platform which could put both tweeting and forum discussion so that "tweets" could get connected, knowledge is automatically aggregated, etc. Also, we don't have to pay Twitter to get the data we generate.

Suggestion: move it under  TCF platform building ideas. Find a temporary solution which could eventually transfer clients to the new platform, by keeping them engaged and aware, and later by creating a migration tool

**Comment by jinaamini** [comment]
We need to define what is objective first and who is our target audience. Twitter works because it's a global community, we can share our voices, ask outside world for help, raise awareness. A new social media should probably target the Iranian audience (abroad and in Iran) and as @IranNewsWatch pointed out not be limited to being a clone of Twitter.  

We can think big and try to become a global community but this road has many challenges and we do not have the time and resources for this.

On a brainstormed thought, we can have revolutionary platform for anyone in the world in the same shoes as us now or in the future. So exactly like @IranNewsWatch pointed out  it can become a large system of resistance including social media, forums, wiki, encyclopedia (ways to fight and resist and how to be secure). But this idea seems to be far away from our reach as we don't have enough resources and time.

**Comment by IranNewsWatch** [comment]
@jinaamini 

Agreed. Basically @armantorkzaban has the same idea but his focus has been on the beung democratic, safe, (pseudo)anonymous, etc.

I think we need a "revolution platform" which is open source, clone-able, deployable for any resistance group.

I think any e-democracy and social collaboration in essense is a resistance/revolution plarform because you are trying to give poeple a platform of free expression and participation in the sort if change, political, reformist, or revolutionary.

If we shift focus with this mindset it might acquire more attention from non-Iranian resistance geoups (RU, Ukraine, Syrians, Chinese, etc). Since we are open source and free to access, we should be safe from "corrupt/vicious intruders".

**Comment by Jiwe-Mobarez** [comment]
Okay so I had this Idea of creating a central repository/hub for digital content that might be useful to the Iranian public inside Iran in case of a Total or Local Internet Outage. As you know, the biggest groups of active participants of the ongoing protests have been young adults and activists. These two main groups have no training in combat, medical procedures, or establishing resistance organizations capable of withstanding regime forces. In short, they are the most vulnerable groups of people to be put in highly critical situations, with an enemy that has shown no sign of humanity and is known to react with extreme violence. 

Most the people participating in the revolution get their information and training from the Internet and social media sites such as twitter. With the Internet gone, many will find themselves in critical situations without being certain what to do. Without a leadership inside the country, I am fearful that many lives will be lost in upcoming chaos should the regime decide to cut off access to the internet.

My Idea is to gather some extensively detailed resources in a handful of essential topics listed bellow with the aim of providing a least bit of critical training if normal ordinary people find themselves needing it. The regime has shown that every time the Internet is disconnected, many regions such as Kurdistan experience brutal violence that is nothing short of a civil war.  Also because we know uploading critical guides and revolutionary resources to local upload centers with a .ir extention will surely be limited by the regime, we need to provide people with the appropriate training material before they lose access.

1- First Aid Books, medical books: If regime forces shoot live rounds at crowds of people like they have done before, many are expected to be hit. In that scenario finding appropriate help and professional medical assistance seems improbable. So people might bring their friends or loved ones back home and not risk going to a regime controlled hospital full of wounded people. 

They are going to need actual resources to perform operations in order to save each other's lives. They also need to be told what medical resources, like band-aids or gauze, they are likely to be needing if an accident befalls them. 

2- Survival Books: Similar to the previous one, If the regime forces surround a city with the aim of breaking the people's spirits, or if fights happen in a region and local supply chains are disrupted, many people might find themselves in a situations that require the knowledge of general survival strategies in urban or rural settings. We need to give them appropriate reading materials to increase their survival chances. 

3- Leadership books, guerrilla warfare tactics, civil disobedience books, etc: With the protests becoming more and more brutal as the internet is shut down and more people are joining the ranks of the protestors, ordinary people are more likely to decide they wanna join the people. These types of people have had no access to resources to teach them how to mobilize, how to lead, how to follow, how to conquer important sites, etc. We need a few good resources for these people to read ass well.

~~~

All the above resources will ideally need to be:

1- Short, Concise, and to the point. They shouldn't be boring.  So info-graphics would be appropriate here. Maybe a few short handbooks in each topic. Another good resource is videos showing people how to do stuff. There are plenty of Youtube videos on these topics. But we have to pay attention to the abysmal internet connections leading to the final outage. So the people providing these videos will have to lower the hard-disk space they take.

2- Preferably translated to Persian, Kurdi, Turki, etc. This is hopefully a nobrainer, since normal people wound't find English books useful in a case of emergency. Videos will need to have subtitles. 

3- Each category needs to be availabe as soon as possible in a ready to download zip folder. We have no time to educate people about wget or curl utilities. We need a final zip file for each category that has everything a person needs + furthur resources if the outages become permanent

4- Be served from multiple mirror sites.

**Comment by armantorkzaban** [comment]
@Jiwe-Mobarez, 
Please go ahead and prepare a list of the educational material you mentioned.
Would you also like to become a member of this task force? if so, follow here: https://github.com/tcfev/task-force-nika/discussions/2
We will help you with that. 

Aside from that, here is an initial list of resources, which we can expand: 
https://github.com/tcfev/iranliberationmovement.com/tree/main/TXT
